roles/translator/src/upstream_sv2/upstream.rs

```diff
@@ -717,25 +724,29 @@ impl ParseUpstreamMiningMessages<Downstream, NullDownstreamMiningSelector, NoRou
     /// Handles the SV2 `OpenExtendedMiningChannelError` message (TODO).
     fn handle_open_mining_channel_error(
         &mut self,
-        _: roles_logic_sv2::mining_sv2::OpenMiningChannelError,
+        m: roles_logic_sv2::mining_sv2::OpenMiningChannelError,
     ) -> Result<roles_logic_sv2::handlers::mining::SendTo<Downstream>, RolesLogicError> {
-        todo!()
+        Ok(SendTo::None(Some(Mining::OpenMiningChannelError(
```

Comment:
`SendTo::None(Some(Mining::OpenMiningChannelError)` is not matched in `match next_message_to_send` this lead to a panic

Reply:
Am I missing something? That case was handled in the line changes above at 449.